staging/src/k8s.io/controller-manager/options/generic.go

```diff
@@ -63,8 +63,8 @@ func (o *GenericControllerManagerConfigurationOptions) AddFlags(fss *cliflag.Nam
 	genericfs.Int32Var(&o.ClientConnection.Burst, "kube-api-burst", o.ClientConnection.Burst, "Burst to use while talking with kubernetes apiserver.")
 	genericfs.DurationVar(&o.ControllerStartInterval.Duration, "controller-start-interval", o.ControllerStartInterval.Duration, "Interval between starting controller managers.")
 	genericfs.StringSliceVar(&o.Controllers, "controllers", o.Controllers, fmt.Sprintf(""+
-		"A list of controllers to enable. '*' enables all on-by-default controllers, 'foo' enables the controller "+
-		"named 'foo', '-foo' disables the controller named 'foo'.\nAll controllers: %s\nDisabled-by-default controllers: %s",
+		"A comma-separated list of controllers to enable. '*' enables all on-by-default controllers, 'foo' enables the controller "+
+		"named 'foo', '-foo' disables the controller named 'foo'.\nAll controllers: %s\nDisabled-by-default controllers: %s. If you want to enable tokencleaner additionally, set '*,tokencleaner'",
```

Comment:
Since below string joins list the "disabled-by-default controllers" (currently bootstrapsigner, tokencleaner), can we instead say something more general that applies to all the disabled-by-default controllers here instead of special casing tokencleaner?

Reply:
If you want to enable a disabled-by-default controller additionally, for example 'foo', please set '*,foo'"

Change to the above, is that ok?

Reply:
That's great.

Reply:
Or just, "'*,foo' enables a disabled-by-default controller."

Reply:
Fixed. Thanks.